index.js

```diff
@@ -4,18 +4,39 @@ const app = express();
 const cors = require('cors');
 
 
-
-
+// middlewares
 app.use(express.json());
 app.use(cors());
 app.use(express.urlencoded({ extended: true }));
 
+//Home page
+app.get("/", (req, res) => {
+    res.send("Welcome to PJMiles API");
+  });
+  
+// routes & API
+const Blog = require('./routes/blogRoutes');
+
+// create blog
+app.post('/blogpost', Blog)
+
+
+
+
+
+
+
+
+
+
 
+// database 
 const database = () => {
     const db = require('./config/mongoose')
     return db
 }
```

Comment:
Why?

Reply:
I wanted to keep the database connection separate

Reply:
Why?